stresstests/test/event_hub_helper.js

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+const EventHubClient = require('@azure/event-hubs').EventHubClient;
+const EventPosition = require('@azure/event-hubs').EventPosition;
+const debug = require('debug')('stresstests:EventHubHelper');
+
+/**
+ * A thenable that can be can be resolved or rejected from outside and records
+ * the time it takes to settle.
+ */
+class Deferred {
+  constructor() {
+    this.startTime = Date.now();
+    this.timeToSettle = null;
+    this.promise = new Promise((resolve, reject) => {
```

Comment:
>promise [](http://example.com/codeflow?start=9&length=7)

Should `this.promise` be an instance variable or a closure (hidden) variable?  If we don't want user's touching it, we should either underscore it or hide it.